operator/src/test/java/oracle/kubernetes/operator/helpers/PodHelperTestBase.java

```diff
@@ -1520,6 +1590,13 @@ public void whenPodCreated_createPodWithOwnerReference() {
     assertThat(getCreatedPod().getMetadata().getOwnerReferences(), contains(expectedReference));
   }
 
+  // TODO: add tests for dynamic updates
```

Comment:
How hard would it be to create these tests? I'm not a fan of leaving a TODO in the code.

Reply:
actually the tests are there, removing TODO